eve/main.yml

```diff
@@ -1553,3 +1556,277 @@ stages:
+          command: |
+            ssh -F $SSH_CONFIG bootstrap --
+            bash -c '
+            IP=$(ip a show eth0 | grep -Po "inet \K[\d.]+") && echo $IP'
+          haltOnFailure: true
+      # AddMe: Add a step to copy Cypress test-suite to node1
+      # No Cypress test-suite exist as of now
+      # Once test-suite become available, we could simply tar archive the
+      # test-suite dir and scp it to node1
+      - ShellCommand:
+          name: Retrieve example-solution ISO image
+          env:
+            BASE_URL: "%(prop:artifacts_private_url)s/example-solution"
+            DEST_DIR: "."
+            SOLUTION_VERSION: "%(prop:example_solution_version)s.iso"
+          command: >
+            curl -s -XGET -o "${DEST_DIR}/example-solution.iso"
+            "${BASE_URL}-${SOLUTION_VERSION}"
+          timeout: 2400
+          haltOnFailure: true
+      - ShellCommand:
+          name: Copy example-solution ISO to bootstrap node
+          command: >
+            scp -F ssh_config ../../../../example-solution.iso
+            bootstrap:/var/tmp/example-solution.iso
+          workdir: build/eve/workers/openstack-multiple-nodes/terraform/
+          haltOnFailure: true
+      - ShellCommand:
+          name: Import example-solution
+          command: >
+            ssh -F ssh_config bootstrap
+            sudo bash
+            /var/tmp/metalk8s/solutions.sh import --archive
+            /var/tmp/example-solution.iso --verbose
+          workdir: build/eve/workers/openstack-multiple-nodes/terraform/
+          haltOnFailure: true
+      - ShellCommand:
+          name: Activate example-solution
+          env:
+            SOLUTION_VERSION: "%(prop:example_solution_version)s"
+          command: >
+            ssh -F ssh_config bootstrap
+            sudo bash
+            /var/tmp/metalk8s/solutions.sh activate --name example-solution
+            --version "${SOLUTION_VERSION}" --verbose
+          workdir: build/eve/workers/openstack-multiple-nodes/terraform/
+          haltOnFailure: true
+      - ShellCommand:
+          name: Create example-solution Environment
+          command: >
+            ssh -F ssh_config bootstrap
+            sudo bash
+            /var/tmp/metalk8s/solutions.sh create-env --name example-environment
+            --verbose
+          workdir: build/eve/workers/openstack-multiple-nodes/terraform/
+          haltOnFailure: true
+      # Taint bootstrap node to allow deployment of example-solution-operator
+      - ShellCommand:
+          name: Taint Bootstrap node before deploying example-solution-operator
+          env:
+            KUBECONF: "--kubeconfig /etc/kubernetes/admin.conf"
+          command: >
+            ssh -F ssh_config bootstrap
+            sudo kubectl ${KUBECONF} taint nodes --all node-role.kubernetes.io/infra:NoSchedule-
+            sudo kubectl ${KUBECONF} taint nodes --all node-role.kubernetes.io/bootstrap:NoSchedule-
```

Comment:
It will not work this way, everything will be put in a single kubectl command.

Reply:
yep it does not